server/util/log/log.go

```diff
@@ -302,6 +303,15 @@ func Infof(format string, args ...interface{}) {
 	log.Info().Msgf(format, args...)
 }
 
+// CtxInfo logs to the INFO log.
+// Logs are enriched with information from the context
+// (e.g. invocation_id, request_id)
+func CtxInfo(ctx context.Context, message string) {
```

Comment:
Can you add the other funcs (CtxError, CtxDebug, etc) for completeness?

Reply:
Done